lib/csv.rb

```diff
@@ -1342,8 +1341,24 @@ def build_header_fields_converter
       accept_nil: true,
     }
     options = @base_fields_converter_options.merge(specific_options)
+    build_converters(@initial_header_converters, options)
+  end
+
+  def writer_fields_converter
+    @writer_fields_converter ||= build_writer_fields_converter
+  end
+
+  def build_writer_fields_converter
+    specific_options = {
+      builtin_converters: Converters,
+    }
+    options = @base_fields_converter_options.merge(specific_options)
+    build_converters(@initial_write_converters, options)
+  end
+
+  def build_converters(initial_converters, options)
```

Comment:
This builds a fields converter from converters. This doesn't build converters.
Do you have more good name?

Reply:
What do you think about `merge_` since it merges the defaults and any custom converter or maybe just call it `fields_converter` since it returns an instance of `FieldsConverter`?

Reply:
I the end I named it `generate_fields_converter` since it is exactly what this method does. Let me know if you think this is a good name :+1:

Reply:
It's better than before. But `build_` and `generate_` may be confused. "generate_" is OK in this pull request. This is a separated issue. I'll consider better name later.